vendor/github.com/openshift/api/operator/v1/types_network.go

```diff
@@ -94,6 +94,10 @@ type DefaultNetworkDefinition struct {
 	// not implemented.
 	// +optional
 	OVNKubernetesConfig *OVNKubernetesConfig `json:"ovnKubernetesConfig,omitempty"`
+
+	// ChainedPlugins is an array of entries to insert as chained plugins for execution
+	// after the default network.
+	ChainedPlugins []string `json:"chainedPlugins,omitempty"`
```

Comment:
I'm not super happy about this API. We should probably give ourselves room to expand. For example, we could also include an explicit tuning configuration, rather than requiring raw CNI.

Reply:
This should say what the strings are supposed to be; eg CNI JSON configuration in string form.

Reply:
OK, changed the API slightly, and added better documentation.